joomla-projects/gsoc22_accessibility-plugin
# Made feedback redirection link & Accessibility Statement available in backend

### Summary of Changes

Added the functionality for feedback redirection link & Accessibility statement. Site admins can select the Menu item where they want to redirect the user to report accessibility-related problems, and also for accessibility statements.

### Testing Instructions

Check that you can select the Menu Items for Accessibility statement and Accessibility Feeedback in the Accessibility plugin backend.

### Actual result BEFORE applying this Pull Request

![Screenshot (435)](https://user-images.githubusercontent.com/54266411/183020867-c603fc99-4d33-4534-a59b-1ac25774207d.png)


### Expected result AFTER applying this Pull Request

![image](https://user-images.githubusercontent.com/54266411/183105544-2ec1dded-f837-4315-81cf-3624035d82dd.png)
